validator-generator/src/main/java/io/avaje/validation/generator/ElementAnnotationContainer.java

```diff
@@ -2,156 +2,131 @@
 
 import static io.avaje.validation.generator.APContext.typeElement;
 import static io.avaje.validation.generator.PrimitiveUtil.isPrimitiveValidationAnnotations;
-import static java.util.stream.Collectors.toMap;
+import static java.util.stream.Collectors.toList;
 
-import java.util.HashMap;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import java.util.Map.Entry;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
+import java.util.stream.Stream;
 
 import javax.lang.model.element.AnnotationMirror;
 import javax.lang.model.element.Element;
 import javax.lang.model.element.ExecutableElement;
-import javax.lang.model.element.VariableElement;
 
 record ElementAnnotationContainer(
     UType genericType,
     boolean hasValid,
-    Map<UType, String> annotations,
```

Comment:
It turns out we never actually supported repeatable annotations. The map key is the annotation type, so when I made the prisms update, all of a sudden I faced errors because map keys must be unique. The map key was the annotation type, which caused the issue.

Reply:
Ah right, and the rest of the changes come from that ... right - thanks!!